packages/gestalt/src/Caret.js

```diff
@@ -2,36 +2,37 @@
 import React, { type Node } from 'react';
 import PropTypes from 'prop-types';
 
+const PATHS = {
+  down:
+    'M 0 4 C 0.69415 4 1.35929 3.7216 1.84646 3.2271 L 4.3752 0.66038 C 5.182 -0.15852 6.5628 -0.224911 7.4592 0.512109 C 7.5161 0.558899 7.5703 0.60839 7.6215 0.66038 L 10.148 3.2248 C 10.6367 3.7208 11.3038 4 12 4',
```

Comment:
@mminute would it be possible to run the original SVG through `svgo`? https://github.com/svg/svgo

That way we can lower the file size of these inline SVGs.